usergroups.go

```diff
@@ -74,25 +74,66 @@ func (api *Client) CreateUserGroupContext(ctx context.Context, userGroup UserGro
 		values["channels"] = []string{strings.Join(userGroup.Prefs.Channels, ",")}
 	}
 
+	if userGroup.TeamID != "" {
+		values["team_id"] = []string{userGroup.TeamID}
+	}
+
 	response, err := api.userGroupRequest(ctx, "usergroups.create", values)
 	if err != nil {
 		return UserGroup{}, err
 	}
 	return response.UserGroup, nil
 }
 
+// DisableUserGroupOption options for the DisableUserGroup and EnableUserGroup method calls.
+type DisableUserGroupOption func(*DisableUserGroupParams)
```

Comment:
👋 thank you for the PR and apologies it took us this long to get to it.

If you're still up for it, I think a few changes are worth making (feel free to let me know if you're no longer interested in adjusting this PR and I can take care of it).

1. Let's rebase first
2. Either
a) change the name so that it's generic enough to use for both Disable and Enable? Having users use a "DisableUserGroupOption" on an EnableUserGroup method is counterintuitive to me.
b) type alias this to an "EnableUserGroupOption" (and "EnableUserGroupParams" as well)